osler/templates/workup/attestablebasicnote_detail.html

```diff
@@ -0,0 +1,45 @@
+{% extends "core/base.html" %}
+
+{% block title %}
+Attestable Basic Note: {{ attestablebasicnote.patient.name }}
+{% endblock %}
+
+{% block header %}
+  <div class="row">
+    <div class="col-md-10">
+      <h2>{{ attestablebasicnote.title }}</h2>
+      <p class="lead">For <a href="{% url 'core:patient-detail' pk=attestablebasicnote.patient.id %}">{{ attestablebasicnote.patient.name }}</a></p>
+    </div>
+    {% if can_modify_attestablebasicnote %}
+    <div class="col-md-1 pull-right">
+    <h1><a href="{% url 'attestable-basic-note-update' pk=attestablebasicnote.id %}">
+      <small><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span></small></a>
+    </h1>
+    </div>
+    {% endif %}
+  </div>
+{% endblock %}
+
+{% block content %}
+<div class="container">
+  <div class="row">
+    <h3>{{ attestablebasicnote.title}}</h3>
```

Comment:
Rather than make two templates that are almost the same other than this Attestation part, can you make a generic one with an "if attestable"? you could define in model like attestable=True kind of like how note_type is defined for Action Item, Followup, etc and many use the same generic html template.

Reply:
smart. I like that